src/errors/runtime/index.js

```diff
@@ -68,6 +70,8 @@ export class APIError extends Error {
         // See https://github.com/DevExpress/testcafe/blob/v1.0.0/src/compiler/test-file/formats/raw.js#L22
         // Also we can't use an ES6 getter for the 'stack' property, because it will create a getter on the class prototype
         // that cannot override the instance property created by the Error parent class.
+        const renderers = this.callsite?.constructor.name === CALLSITE_RECORD_CLASS_NAME ? renderersRecord : renderersCommand;
```

Comment:
Move this expression to the `getRenderes(callsite)` utility function and use it in all places.